spec/factories/trackings.rb

```diff
@@ -10,5 +10,9 @@
     ndc_number            '12345678910'
     shipped_date          Time.parse('Thu, 12 Oct 2016 00:00:00 EDT').in_time_zone
     delivery_service      'UPS'
+
+    trait :oldest do
+      shipped_date Time.parse('Thu, 12 Oct 2016 00:00:00 EDT').in_time_zone - 1.year
```

Comment:
Any way for this to be more relative to the default `shipped_date`?

Reply:
You mean something similar in function to `shipped_date = shipped_date - 1.year`?

Reply:
Yup.